www/js/lib/uiUtil.js

```diff
@@ -227,12 +228,88 @@ define([], function() {
             '/': '&#x2F;',
             '`': '&#x60;',
             '=': '&#x3D;'
-          };
-          string = String(string).replace(/[&<>"'`=/]/g, function (s) {
-            return escapechars[s];});
-          return string;
+        };
+        string = String(string).replace(/[&<>"'`=/]/g, function (s) {
+            return escapechars[s];
+        });
+        return string;
+    }
 
+    /**
+     * Removes the animation effect between various sections
+     */
+    function removeAnimationClasses() {
+        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
+        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
+        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+    }
+    
+    /**
+     * Adds the slide animation between different sections
+     * 
+     * @param {String} name_of_section It takes the name of the section to which the animation is to be added
+     * 
+     */
+    function animateToSection(section) {
+        var sectionsMap = {'home':0,'config':1,'about':2}
+        switch(sectionsMap[section]) {
```

Comment:
It's not what I was thinking about. Sorry I should have been more specific.
The 3 cases of this swithc are very similar and could be merged. Instead of using ${'configuration'}, you could use ${section} to show it, and use a loop on the other 2 sections, to hide them. Only one generic code to handle the 3 (or more) sections

Reply:
Yes , I have tried doing that but if you look at different switch cases closely, you will observe that each switch case has completely different set of statements and minor similarities, So it's not possible to change it.
Also one of the reasons is also that the sections are not just hidden, their classes are changed and that also every time different ones.

Reply:
OK never mind.
So I'd rather revert to the previous version without the switch/case. The if/elseif/elseif looks more readable to me as you directly see the section name.